Makefile

```diff
@@ -47,7 +47,12 @@ build-revision:
 	gox -verbose ${GO_LDFLAGS} -os="${BUILD_OS}" -arch="${BUILD_ARCH}" -output="${DIST_DIR}/${REVISION}/deis-${REVISION}-{{.OS}}-{{.Arch}}" .
 
 # This is supposed to be run within a docker container
-build-tag:
+build-stable:
+	$(eval GO_LDFLAGS = -ldflags '-X ${repo_path}/version.Version=${GIT_TAG}')
+	gox -verbose ${GO_LDFLAGS} -os="${BUILD_OS}" -arch="${BUILD_ARCH}" -output="${DIST_DIR}/deis-stable-{{.OS}}-{{.Arch}}" .
+
+# This is supposed to be run within a docker container
+build-tag: build-stable
```

Comment:
@Joshua-Anderson good point.  also a good excuse to have a go at parallelization in the workflow-cli-release job (will write that PR up now)... updated.